sdk/basyx/objectstore.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2023 the Eclipse BaSyx Authors
+#
+# This program and the accompanying materials are made available under the terms of the MIT License, available in
+# the LICENSE file of this project.
+#
+# SPDX-License-Identifier: MIT
+"""
+This module implements Registries for the AAS, in order to enable resolving global
+:class:`Identifiers <basyx.aas.model.base.Identifier>`; and mapping
+:class:`Identifiers <basyx.aas.model.base.Identifier>` to :class:`~basyx.aas.model.base.Identifiable` objects.
```

Comment:
I think these references are broken. 
I'm also not sure how one would reference objects from another package (aas-core3-Python) in Sphinx. For now, I think we should just write `Identifier` and add an issue to fix the sphinx autodocs, when we add it.

Reply:
I went through the object_store.py file and replaced all external references. Should we just completely remove these autodoc class references for now or only those that break as they point to external objects?

Reply:
I think we should only remove those pointing to external objects. Otherwise we'll have more work when reintroducing autogenerated documentation, which we plan to do.

Reply:
Okay then this should be fixed. I removed all he links pointinf to basyx.aas... but left those pointing to local objects. So I think the current state should compile fine, I didnt manage to test it unfortunately